python-sdk/tests/test_example_dags.py

```diff
@@ -92,3 +92,22 @@ def test_example_dynamic_task_map_dag(session, dag_id):
     if dag is None:
         raise NameError(f"The DAG with dag_id: {dag_id} was not found")
     wrapper_run_dag(dag)
+
+
+@pytest.mark.skipif(
+    airflow.__version__ < "2.4.0", reason="Require Airflow version >= 2.4.0"
+)
+@pytest.mark.parametrize(
+    "dag_id",
+    [
+        "example_dataset_producer",
```

Comment:
Do we need to test that consumer DAG is triggered ?

Reply:
Yes , and this will need >=2.4.0

Reply:
I think we can skip testing this since it's something specific to airflow and not AstroSDK.

Reply:
Just wondering how can we ensure that if a developer changes the code relevant to mapping the Datasets to Astro-SDK, that it is not breaking any functionality?